skt/reporter.py

```diff
@@ -308,6 +315,9 @@ def __getjobresults(self):
                     # tasks.
                     if task_result != 'Pass':
```

Comment:
Not asking to change this now, but I think it is better to streamline conditions when you can: e.g. remove extra negations, so they're easier to read and reason about. In this case, since you added the second branch, it was a good opportunity to do an equals comparison instead, and switch the branches. I.e.:

    if task_result == 'Pass':
        test_name = task_node.attrib.get('name')
        passed_tasks.append(test_name)
    else:
        failed_tasks.append(task_node)